example/tools/lint/linters.bzl

```diff
@@ -113,8 +114,14 @@ clang_tidy_global_config = lint_clang_tidy_aspect(
 )
 
 spotbugs = lint_spotbugs_aspect(
-    binary = "@@//tools/lint:spotbugs",
-    exclude_filter = "@@//:spotbugs-exclude.xml",
+    binary = Label("@//tools/lint:spotbugs"),
+    exclude_filter = Label("@//:spotbugs-exclude.xml"),
 )
 
 spotbugs_test = lint_test(aspect = spotbugs)
+
+keep_sorted = lint_keep_sorted_aspect(
+    binary = Label("@com_github_google_keep_sorted//:keep-sorted"),
```

Comment:
Was i missing `Lablel()` so it would make apparent -> canonical?
I thought I could just write the apparent as a string and it would change it as necessary.

Reply:
The problem is which context strings are evaluated to labels.
When you pass a string to a macro or function, it is evaluated in the receiving context, in this case that's in rules_lint which won't have this apparent repository.
The explicit Label() call here ensures it's evaluated in this repository context.